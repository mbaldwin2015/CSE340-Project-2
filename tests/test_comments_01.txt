// Comment on a line by itself
xyz, it, c4;

hello {
  public: xyz, it;
  private: c4, dd;
  
  world1 {
    xyz = it;
    c4 = dd;
  }
  world2 {
    public: xyz, c4;

    xyz = it;
    c4 = dd;
  }
}
