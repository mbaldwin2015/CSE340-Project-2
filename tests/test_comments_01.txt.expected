hello.xyz = hello.it
::c4 = ?.dd
world2.xyz = hello.it
world2.c4 = ?.dd
